src/core/storage/fileio/file_download_cache.cpp

```diff
@@ -60,12 +60,8 @@ std::string file_download_cache::get_file(const std::string& url) {
   int status; bool is_temp;
   std::tie(status, is_temp, localfile) = download_url(url);
   if (status) {
-#ifdef TC_ENABLE_REMOTEFS
```

Comment:
should we keep this prompt? I think it's kind of informational,

Reply:
The logic has changed -- it's now one macro, TC_DISABLE_REMOTEFS.  and the original logic was wrong.